src/components/Dashboard/Analytics/AnalyticsUtils.js

```diff
@@ -0,0 +1,121 @@
+// Barcharts
+
+export const barChartOptions = {
+    plugins: {
+        legend: {
+            position: "top",
+            display: false,
+        },
+        title: {
+            display: false,
+            text: "Activity",
+        },
+    },
+    maintainAspectRatio: false,
+    responsive: true,
+};
+
+export const mainBarChartDatasets = [
+    {
+        label: "Contributions days",
+        data: [345, 567, 765, 453, 676, 876, 564, 234, 440, 280, 440, 280],
+
+        borderColor: "rgb(128, 128, 128)",
+        backgroundColor: [
+            "rgb(255, 99, 132)",
+            "rgb(255, 106, 9)",
+            "rgb(54, 162, 235)",
+            "rgb(0, 47, 225)",
+            "rgb(255, 205, 86)",
+            "rgb(255, 99, 132)",
+            "rgb(255, 106, 9)",
+            "rgb(54, 162, 235)",
+            "rgb(0, 47, 225)",
+            "rgb(255, 205, 86)",
+            "rgb(169, 169, 169, 0.1)",
+            "rgb(169, 169, 169, 0.1)",
+        ],
+        hoverOffset: 4,
+    },
+];
+
+export const smallBarChartDatasets = [
+    {
+        label: "Contributions days",
+        data: [345, 567, 765, 453, 676, 876, 564, 234, 440, 280, 440, 280],
+
+        borderColor: "rgb(128, 128, 128)",
+        backgroundColor: [
+            "rgb(255, 99, 132)",
+            "rgb(255, 106, 9)",
+            "rgb(54, 162, 235)",
+            "rgb(0, 47, 225)",
+            "rgb(255, 205, 86)",
+            "rgb(169, 169, 169, 0.1)",
+            "rgb(169, 169, 169, 0.1)",
+            "rgb(169, 169, 169, 0.1)",
+            "rgb(169, 169, 169, 0.1)",
+            "rgb(169, 169, 169, 0.1)",
+            "rgb(169, 169, 169, 0.1)",
+            "rgb(169, 169, 169, 0.1)",
+            "rgb(169, 169, 169, 0.1)",
+        ],
+        hoverOffset: 4,
+    },
+];
+
+export function displayMonths(count) {
```

Comment:
good, but make it more dynamic ,example: if I would like a range from Mar to Jun ...